src/shared/eyetracking.py

```diff
@@ -529,13 +529,15 @@ def read_pl_data(fname):
 
 
 def fixation_dot(win, **kwargs):
-    radius = kwargs.pop('radius', 30)
+    #radius = kwargs.pop('radius', 30)
```

Comment:
to cleanup

Reply:
ok! I was trying out params and didn't want to lose the "original" size